src/main/java/com/crafter/springboottechiteasy/models/Television.java

```diff
@@ -0,0 +1,63 @@
+package com.crafter.springboottechiteasy.models;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "Televisions")
+public class Television {
+
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    Long id;
```

Comment:
Deze zijn nu package private, het is gebruikelijk om dit private te maken. en dan de getters and setters te gebruiken

Reply:
Zag ik net ook, dus inmiddels is de private toegevoegd ;)